README.rst

```diff
@@ -19,9 +19,11 @@ A Python Framework For Using HAR Files To Analyze Web Pages.
 Overview
 --------
 
-The haralyzer module contains two classes for analyzing web pages based
+The haralyzer module contains three classes for analyzing web pages based
 on a HAR file. ``HarParser()`` represents a full file (which might have
-multiple pages), and ``HarPage()`` represents a single page from said file.
+multiple pages). ``HarPage()`` represents a single page from said file.
+``HarEntry()`` represents an entry in a ``HarPage()`` there are are multiple entries per page.
```

Comment:
The grammar feels awkward here. Thinking one of the following:

"``HarEntry()`` represents an entry in a ``HarPage()``. There are are multiple entries per page."
"``HarEntry()`` represents an entry in a ``HarPage()``, and there are are multiple entries per page."